src/dataExplorer/context/persistance.tsx

```diff
@@ -152,7 +161,7 @@ export const PersistanceProvider: FC = ({children}) => {
   const [resource, setResource] = useSessionStorage('dataExplorer.resource', {
     type: 'scripts',
     flux: '',
-    language: LanguageType.FLUX,
+    language: isIoxOrg ? LanguageType.SQL : LanguageType.FLUX,
```

Comment:
So on load of the QxBuilder, if iox org, will go to the SQL text. 👍🏼

Reply:
Yup, that's the intent